extensions/amp-imgur/0.1/validator-amp-imgur.protoascii

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-imgur
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-imgur extension .js script"
+  extension_spec: {
+    name: "amp-imgur"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+  spec_name: "amp-imgur extension .js script"
```

Comment:
Everything from `spec_name` down in this `tag` is not necessary as it is handled by `extension_spec`. To be clear, please remove:

```
  spec_name: "amp-imgur extension .js script"
  satisfies: "amp-imgur extension .js script"
  requires: "amp-imgur"
  mandatory_parent: "HEAD"
  unique: true
  extension_unused_unless_tag_present: "amp-imgur"
  attrs: {
    name: "async"
    mandatory: true
    value: ""
  }
  attrs: {
    name: "custom-element"
    mandatory: true
    value: "amp-imgur"
    dispatch_key: true
  }
  attrs: { name: "nonce" }
  attrs: {
    name: "src"
    mandatory: true
    value_regex: "https://cdn\.ampproject\.org/v0/amp-imgur-(latest|0\.1).js"
  }
  attrs: {
    name: "type"
    value: "text/javascript"
  }
  cdata: {
    blacklisted_cdata_regex: {
      regex: "."
      error_message: "contents"
    }
  }
  spec_url: "https://www.ampproject.org/docs/reference/components/amp-imgur"
```